config/validator/service/proposer-settings.go

```diff
@@ -0,0 +1,35 @@
+package validator_service_config
+
+import (
+	"github.com/ethereum/go-ethereum/common"
+	field_params "github.com/prysmaticlabs/prysm/config/fieldparams"
+)
+
+// ProposerSettingsConfig is the struct representation of the JSON or YAML payload set in the validator through the CLI.
+// ProposeConfig is the map of validator address to fee recipient options all in hex format.
+// DefaultConfig is the default fee recipient address for all validators unless otherwise specified in the propose config.required.
+type ProposerSettingsConfig struct {
```

Comment:
I think we can lose the suffix Config since the package name already has config.  Maybe `ProposerSettingsCli`?

Reply:
trying out naming it payload instead, let me know what you think